python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml

```
@@ -1425,6 +1429,175 @@ keys:
+                trunk:
+                  type: str
+                  description: |-
+                    Warning: This should not be combined with `ethernet_interfaces[].phone.trunk`
+                  valid_values: ["tagged", "tagged phone", "untagged", "untagged phone"]
+            mode:
+              type: str
+              valid_values: ["access", "dot1q-tunnel", "trunk", "trunk phone"]
+              description: |-
+                Warning: This should not be combined with `ethernet_interfaces[].mode`
+            pvlan_mapping:
+              type: str
+              convert_types:
+                - int
+              description: |-
+                Secondary VLAN IDs of the private VLAN mapping.
+                Warning: This should not be combined with `ethernet_interfaces[].pvlan_mapping`.
+            vlan_translations:
+              description: |-
+                Warning: This should not be combined with `ethernet_interfaces[].vlan_translations`.
+              type: list
+              primary_key: direction
+              items:
+                type: dict
+                keys:
+                  from:
+                    type: str
+                    convert_types:
+                      - int
+                    description: VLAN ID or range of VLAN IDs to map from.
+                    required: true
+                  to:
+                    type: int
+                    convert_types:
+                      - str
+                    description: VLAN ID to map to.
+                    required: true
+                  direction:
+                    type: str
+                    valid_values: ["in", "out", "both"]
+                    default: "both"
+            vlan_forwarding_accept_all:
+              type: bool
+            backup_link:
+              type: dict
+              keys:
+                interface:
+                  type: str
+                  description: Backup interface. Example - Ethernet4, Vlan10 etc.
+                prefer_vlan:
+                  type: int
+                  convert_types:
+                    - str
+                  min: 1
+                  max: 4094
+                  description: VLANs to carry on the backup interface.
+            backup:
+              type: dict
+              description: The `backup_link` is required to set this config.
+              keys:
+                dest_macaddr:
+                  type: str
+                  description: Destination MAC address for MAC move updates.
+                initial_mac_move_delay:
+                  type: int
+                  convert_types:
+                    - str
+                  description: Initial MAC move delay in milliseconds.
+                  min: 0
+                  max: 65535
+                mac_move_burst:
+                  type: int
+                  convert_types:
+                    - str
+                  description: Size of MAC move bursts in milliseconds.
```

```suggestion
                  description: Size of MAC move bursts.
```

---

it is in milliseconds - 
```
s1-leaf2(config-if-Et2)#switchport backup mac-move-burst ?
  <0-65535>  Milliseconds
```

---

but it is the `Size of MAC move bursts.` so I am not sure if it should be in `milliseconds`.
@ClausHolbechArista @gmuloc ?

---

Let's keep milliseconds out of this. I don't think it is milliseconds. I have submitted a bug to fix the text in EOS.
```suggestion
                  description: Size of MAC move bursts.
```

---

Fixed